install.sh

```diff
@@ -31,8 +31,8 @@ else
 	deno_uri="https://github.com/denoland/deno/releases/download/${1}/deno_${os}_x64.gz"
 fi
 
-deno_install=${DENO_INSTALL:-$HOME/.deno}
-bin_dir="${deno_install}/bin"
+deno_install=${DENO_INSTALL:-$HOME/.local}
+bin_dir=$deno_install/bin
```

Comment:
Maybe leave the quotation marks? Without quotation marks maybe this breaks in the case that $deno_install has spaces in it

Reply:
This is a variable assignment. What you say is true for e.g. running a command, but variable assignments have their own rules:

At least in bash or zsh, assigning variables like this will work, AFAIK even with consecutive spaces.

So since the shebang is /bin/sh, if that links to bash or zsh it’ll work. Idk how it is for other POSIX shells.